### ISSUE TITLE
Hello, Thank you very much for your Git

### ISSUE THREAD

**Issue by yeyuxmf** [liulei01/DRBox#2]
What is the data storage format of the label TXT file？For example, there are multiple targets in an image, and how it corresponds to the label TXT file. I can't open the downloaded data and show the damage.Thank you.

**Comment by liulei01** [comment]
In folder "train_data", each image file has a corresponding .rbox file. For example, an image file 1.tif would be labeled by 1.tif.rbox. Supposing that there are two targets in this image, the rbox file should have two lines and each line corresponds to one target. For example, it may looks like:

128.0 117.0 50.0 60.0 1 16.5
262.0 152.0 87.0 90.0 1 250.0

The 5 numbers in each line are center_x, center_y, width, height, label, angle, respectively.
To build a LMDB database, you should use a list file to indicate the relationship between each image file and its rbox file. For example, in trainval.txt, you should write:

1.tif 1.tif.rbox
2.tif 2.tif.rbox

if you have two images for training.

**Comment by liulei01** [comment]
The downloaded data should be extracted using command cat data.tar.gz*>data.tar.gz first to merge each subfile together.

**Comment by yeyuxmf** [comment]
Thank you very much for your help.I wish you a good mood every day.

**Comment by jeejeelee** [comment]
@liulei01  your dataset is so big for me, can you upload smaller dataset( eg. about 2G ),I just want to learn  the process of training with your dataset thank you ~~~

**Comment by liulei01** [comment]
@jeejeeli Do you have difficulty downloading them or training with all of them? If the latter, you can simply delete lines in trainval.txt to decrease the number of training samples.

**Comment by jeejeelee** [comment]
@liulei01  thanks for your answer， it is difficult for me  to download your data, but  now I have downloaded your dataset using VIP ,hhh~

**Comment by pfchenstuxidianedncn** [comment]
Hi, I download the dataset from the network disk, and I also use command cat data.tar.gz*>data.tar.gz to merge each subfile together. But I get an error when I unzip the data.tar.gz use the command "tar -zxvf data.tar.gz". Which command should I use to unzip the tar file, thank you!

**Comment by pfchenstuxidianedncn** [comment]
Have you download the datasets and extracted them? I have some problem when I extracting them.@huang229
![_2018-01-16_16-03-40](https://user-images.githubusercontent.com/35252396/34977790-e0835950-fad6-11e7-8eca-eb7ba2bf0ffc.png)

**Comment by huang229** [comment]
@pfchenstuxidianedncn   command: cat data.tar.gz*>data.tar.gz

**Comment by huang229** [comment]
@pfchenstuxidianedncn ubuntu system

**Comment by yeyuxmf** [comment]
@pfchenstuxidianedncn  ubuntu system   command: cat data.tar.gz*>data.tar.gz

**Comment by pfchenstuxidianedncn** [comment]
Yes, I have already carried out this command, but I have problem when I extract the "data.tar.gz". How can I solve this problem.

**Comment by cyrilli** [comment]
Hi, I managed to compile and train your model on Airplane data, and I am about to build my own dataset. I am wondering what is BAIYUNJICHANG_Level_19_0.tif.pos.txt in DRBox/data/Airplane/train_data. Is this file needed to create a dataset?
